src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
+import Bookstore from './components/Bookstore';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="book-app">
+      <Bookstore />
     </div>
```

Comment:
- [ ] You should integrate the `react-router-dom` and configure your routes so you have two routes: Bookstore and Categories